cmd/invoke.go

```diff
@@ -14,11 +14,11 @@ import (
 )
 
 func init() {
-	root.AddCommand(NewEmitCmd(newEmitClient))
+	root.AddCommand(NewInvokeCmd(newInvokeClient))
 }
 
 // create a fn.Client with an instance of a
-func newEmitClient(cfg emitConfig) (*fn.Client, error) {
+func newInvokeClient(cfg emitConfig) (*fn.Client, error) {
```

Comment:
Don't we want to rename `emitConfig -> invokeConfig` and the same for `emit-` properties etc? So there is not this mixed naming in this file.